### DISCUSSION TITLE
unattended install ubuntu desktop 20.04.3 amd64 problems

### DISCUSSION THREAD

**Discussion by tazmaniadevil** [BurntSushi/ripgrep#2037] [General] [answered]
everything went like the unattended steps you must take like a preseedfile en a isolinux.cfg .
but my problem is now the boot is oke , **till the installation type**. normally it must boot till the desktop .
i just want to ask what the fault is because i need three more steps to make it work and thats the installaion typ and then it give me the hour region and then the root name and password steps and then everything gos normal , but and unattended dont work like that . can somebody help me please thanx . because it begins too boot by the installation type 
                                                                                                                               the hour region 
                                                                                                                               the root pasword en username
but i want dismiss those three last steps ; is it maybe a fault in my preseedfile .

thx,

**Comment by BurntSushi** [comment, marked as answer]
Sorry, but this reads like gibberish to me. If this is a sincere attempt to get help, you might need someone to help translate your request to English.

**Reply by tazmaniadevil** [reply]
haha okey , i will try. When i use my unattended ubuntu 20.04-3 desktop.iso for in a vm, it boots up all the way but only up to the steps of installation type and location timezone and root bars and passwords.

my question is that I normally have to land directly on the desktop with that update window; but now I have to manually perform those 3 steps and then my output is ready.

my question was whether this is due to my preseed file or to my txt.cfg? or something else in my files.

i hope my english was ok. Thanks in advance for the feedback,

**Reply by BurntSushi** [reply]
It seems a little more coherent, but I don't see what this has to do with ripgrep. I think you're in the wrong place.
